package/tests/test_client.py

```diff
@@ -1,13 +1,63 @@
 import unittest
+import os
 from aistore.client.api import Client
+from aistore.client.msg import Bck
+from aistore.client.const import PROVIDER_AIS
 
 
-class TestStringMethods(unittest.TestCase):
+class TestBasicPutGet(unittest.TestCase):
     def test_list_objects(self):
```

Comment:
I see this test is also checking `get_objects` and incidentally other APIs. I feel it makes to move `get_objects` check here into a different test.

Reply:
Separated bucket and object tests